<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="22267" started-at="2021-12-08T16:36:28Z" finished-at="2021-12-08T16:36:50Z">
    <groups>
      <group name="SmokeTest">
        <method signature="ClickOnSaveAndAllTheFillSareSaveOrNot.cickOnSaveAndAllTheFillSareSaveOrNot()[pri:0, instance:com.crm.SDET26.Opportunity.ClickOnSaveAndAllTheFillSareSaveOrNot@42e26948]" name="cickOnSaveAndAllTheFillSareSaveOrNot" class="com.crm.SDET26.Opportunity.ClickOnSaveAndAllTheFillSareSaveOrNot"/>
      </group> <!-- SmokeTest -->
    </groups>
    <test name="Default test" duration-ms="22267" started-at="2021-12-08T16:36:28Z" finished-at="2021-12-08T16:36:50Z">
      <class name="com.crm.SDET26.Opportunity.ClickOnSaveAndAllTheFillSareSaveOrNot">
        <test-method status="PASS" signature="dataBaseConnection()[pri:0, instance:com.crm.SDET26.Opportunity.ClickOnSaveAndAllTheFillSareSaveOrNot@42e26948]" name="dataBaseConnection" is-config="true" duration-ms="8" started-at="2021-12-08T16:36:28Z" finished-at="2021-12-08T16:36:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- dataBaseConnection -->
        <test-method status="PASS" signature="browserLaunch()[pri:0, instance:com.crm.SDET26.Opportunity.ClickOnSaveAndAllTheFillSareSaveOrNot@42e26948]" name="browserLaunch" is-config="true" duration-ms="3591" started-at="2021-12-08T16:36:28Z" finished-at="2021-12-08T16:36:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- browserLaunch -->
        <test-method status="PASS" signature="login()[pri:0, instance:com.crm.SDET26.Opportunity.ClickOnSaveAndAllTheFillSareSaveOrNot@42e26948]" name="login" is-config="true" duration-ms="3799" started-at="2021-12-08T16:36:31Z" finished-at="2021-12-08T16:36:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="FAIL" signature="cickOnSaveAndAllTheFillSareSaveOrNot()[pri:0, instance:com.crm.SDET26.Opportunity.ClickOnSaveAndAllTheFillSareSaveOrNot@42e26948]" name="cickOnSaveAndAllTheFillSareSaveOrNot" duration-ms="12932" started-at="2021-12-08T16:36:35Z" finished-at="2021-12-08T16:36:48Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.crm.SDET26.Opportunity.ClickOnSaveAndAllTheFillSareSaveOrNot.cickOnSaveAndAllTheFillSareSaveOrNot(ClickOnSaveAndAllTheFillSareSaveOrNot.java:72)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cickOnSaveAndAllTheFillSareSaveOrNot -->
        <test-method status="PASS" signature="logout()[pri:0, instance:com.crm.SDET26.Opportunity.ClickOnSaveAndAllTheFillSareSaveOrNot@42e26948]" name="logout" is-config="true" duration-ms="961" started-at="2021-12-08T16:36:48Z" finished-at="2021-12-08T16:36:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:com.crm.SDET26.Opportunity.ClickOnSaveAndAllTheFillSareSaveOrNot@42e26948]" name="closeBrowser" is-config="true" duration-ms="774" started-at="2021-12-08T16:36:49Z" finished-at="2021-12-08T16:36:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method status="PASS" signature="disconnectDB()[pri:0, instance:com.crm.SDET26.Opportunity.ClickOnSaveAndAllTheFillSareSaveOrNot@42e26948]" name="disconnectDB" is-config="true" duration-ms="1" started-at="2021-12-08T16:36:50Z" finished-at="2021-12-08T16:36:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- disconnectDB -->
      </class> <!-- com.crm.SDET26.Opportunity.ClickOnSaveAndAllTheFillSareSaveOrNot -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
